tests/storybook.ts

```diff
@@ -4,9 +4,14 @@ import { RunOptions } from '../types'
 export async function test(options: RunOptions) {
 	await runInRepo({
 		...options,
-		repo: 'storybookjs/builder-vite',
-		branch: 'main',
-		build: 'prepublish',
-		test: ['build-examples', 'test-ci']
+		repo: 'storybookjs/repro-templates-temp',
+		dir: 'storybook7',
+		packageDir: 'react-vite/default-js/after-storybook',
+		branch: 'next',
+		build: ['yarn install', 'build-storybook'],
+		test: [
+			'yarn add --dev @storybook/test-runner http-server concurrently',
```

Comment:
there is beforeTest but ideally those are installed in the repo already

Reply:
Unfortunately they're not.  We're using this repo here in a bit of an unintended manner.  It's used in storybook's CI process, which doesn't need these to be installed in the projects themselves.

Reply:
can't you use storybook repo with the subdir option? thought that's what that was about

Reply:
I can use the storybook repo, yes.  But that doesn't have these dependencies, which are being added here _just_ for this vite CI job.